stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -435,15 +446,38 @@ public class MessageListViewModel(
         val messageMode = messageMode
 
         if (messageMode is MessageMode.MessageThread) {
-            threadMessagesState = threadMessagesState.copy(isLoadingMore = true)
-            chatDomain.threadLoadMore(channelId, messageMode.parentMessage.id, messageLimit)
```

Comment:
Shouldn't we deprecate `chatDomain.threadLoadMore`?

Reply:
As Oleg said to me, we shouldn't deprecate as there is no public mechanism to thread load more until we release OfflinePlugin.